app/forms/conversion/create_project_form.rb

```diff
@@ -41,6 +42,12 @@ def initialize(params = {})
     super(params)
   end
 
+  CASEWORKER_TEAM_RESPONSES = [OpenStruct.new(id: true, name: I18n.t("yes")), OpenStruct.new(id: false, name: I18n.t("no"))]
```

Comment:
Same with this, as this is only for `voluntary` does it make more sense delcared there?